config/initializers/loadbalancer_ips.rb

```diff
@@ -0,0 +1,13 @@
+aws_ips_file_path = Rails.root.join("config/aws_ips.json")
+
+# File should always exist in prod/stag but allow testing this in dev mode if file exists
+if Rails.env.production? or Rails.env.staging? or File.exists?(aws_ips_file_path)
+  json = File.read(aws_ips_file_path)
```

Comment:
Yes, but worst-case that should just prevent a deploy, which is probably better than this breaking and us storing ELB IPs again without getting notified.